### PR TITLE
Add a check so you can run the app locally

### PR DESCRIPTION
**Pull request by sarthaktexas**

### CODE REVIEW

File: frontend/layouts/dashboard.tsx [hunk 1]
```diff
@@ -125,7 +125,7 @@ export default function DashboardLayout({
   const { data: user, error: userError } = useSWR("/users/me", fetchApi);
 
   useEffect(() => {
-    if (userError) {
+    if (userError && NODE_ENV !== "development") {
```
Comment (cjdenio) [inline frontend/layouts/dashboard.tsx]:
```suggestion
    if (userError && process.env.NODE_ENV !== "development") {
```

Comment (jasonappah) [reply frontend/layouts/dashboard.tsx]: When using `next dev` the `NODE_ENV` is automatically set so this should be good.

Comment (jasonappah) [reply frontend/layouts/dashboard.tsx]: Hold on a sec, it's meant to be process.env.NODE_ENV

**Review by cjdenio — CHANGES_REQUESTED** [review]
See my suggested change

**Comment by sarthaktexas** [comment]
oh shit i'm so sorry @cjdenio thanks for catching that

**Review by jasonappah — APPROVED** [review]
LGTM

**Review by anirudhb — CHANGES_REQUESTED** [review]
lgtm, waiting on Caleb's nit

**Review by cjdenio — APPROVED** [review]
lgtm

**Review by anirudhb — APPROVED** [review]

**Pull request merged**